example/src/hooks/useIterableApp.tsx

```diff
@@ -134,12 +134,18 @@ export const IterableAppProvider: FunctionComponent<
 
     const fn = getIsEmail(id) ? Iterable.setEmail : Iterable.setUserId;
 
-    fn(id);
+    let token;
+
+    if (process.env.ITBL_IS_JWT_ENABLED === 'true' && process.env.ITBL_JWT_SECRET) {
+      token = await getJwtToken();
+    }
+
+    fn(id, token);
```

Comment:
JWT related things can be made a separate PR if it needs attention outside scope of Embedded Messaging.